cookbooks/new_ping.md

```diff
@@ -0,0 +1,45 @@
+Got some new data you want to send to us? How in the world do you send a new ping? Follow this guide to find out.
+
+Write Your Questions
+--------------------
+Do not try and implement new pings unless you know specifically what questions you're trying to answer. General questions about "How do users use our product?" won't cut it - these need to be specific, concrete asks that can be translated to data points. This will also make it easier down the line as you start data review.
```

Comment:
In the style guide that does not exist yet, I'm going to suggest breaking lines at 100 characters or at a sentence break to make code review simpler. 

No change needed here.